client/src/BidderBiddingList.js

```diff
@@ -1,23 +1,36 @@
 import { useEffect, useState } from "react";
-import { get } from "./TenderClient";
+import { get, post } from "./TenderClient";
 
 const BidderBiddingList = () => {
 	const [loading, setLoading] = useState(true);
 	const [bidderList, setBidderList] = useState([]);
 	const [errorMsg, setErrorMsg] = useState(null);
+	const [currentPage, setCurrentPage] = useState(1);
+	const [totalPages, setTotalPages] = useState(1);
+
+	const fetchBidderBids = async (page) => {
+		try {
+			const data = await get(`api/bidder-bid?page=${page}`);
+			setLoading(false);
+			setBidderList(data.results);
+			setTotalPages(data.pagination.totalPages);
+		} catch (error) {
+			setErrorMsg(error.message);
```

Comment:
The endpoint `/bidder-bid` doesn't return any `message` when it returns 500
```suggestion
			setErrorMsg("Server error");
```